### PR TITLE
update readme for todoMVC apps

### PR DESCRIPTION
**Pull request by flashdesignory**
The current readme file for the todoMVC apps is not up-to-date anymore and doesn't represent the workloads included.

Hoping the updated readme will answer the following:

1. What is the workload app
2. What are we testing with this specific app
3. How are we testing it.

Notes:

- Best to look at the readme file directly, instead of the diff.
- added images to repo for the readme
- removed learn.json (that doesn't have any purpose here I believe)

### CODE REVIEW

**Review by addyosmani — APPROVED** [review]
Thanks for the updates! Fwiw, this LGTM.

**Review by housseindjirdeh — APPROVED** [review]
LGTM

File: resources/todomvc/readme.md [hunk 1]
```diff
@@ -1,142 +1,30 @@
...
 
-Everything in this repo is MIT License unless otherwise specified.
+The test consist of the following steps:
 
-MIT © Addy Osmani, Sindre Sorhus, Pascal Hartig, Stephen Sawchuk.
+- Wait for the “add task” input field to be present in the DOM
+- Add a task
+- Check off a task
+- Delete a task
+- Repeat steps 2 - 4 a set number of times (100 by default).
```
Comment (bgrins) [inline resources/todomvc/readme.md:30]: Nit: This is a little ambiguous between whether it's repeating the steps internally (doing step 2 100 times, ...) or across steps (doing step 2-4 100 times). My understanding is it's the former, and so I might rephrase this as something like:

```
- Wait for the “add task” input field to be present in the DOM
- Add 100 tasks
- Check off all tasks
- Delete all tasks
```

File: resources/todomvc/readme.md [hunk 2]
```diff
@@ -1,142 +1,30 @@
...
-
-## Getting Involved
+![screenshot](./screenshot.png)
 
-Whilst we enjoy implementing and improving existing Todo apps, we're always interested in speaking to framework authors (and users) wishing to share Todo app implementations in their framework/solution of choice.
+## What are we testing
 
-Check out our [contribution docs](contributing.md) for more info.
+- Stress-test DOM manipulations of a repeated action.
+- Impact of JavaScript version releases.
```
Comment (bgrins) [inline resources/todomvc/readme.md:16]: What is this referring to? I'm guessing language features, like the VanillaJS-ES2015 test?

**Review by bgrins — APPROVED** [review]
Thanks Thorsten, I think this is a good pattern to use for other tests as well

File: resources/todomvc/readme.md (hunk 1 above)
Comment (flashdesignory) [reply resources/todomvc/readme.md:30]: I'm referring to this:
https://github.com/WebKit/Speedometer/blob/main/resources/tests.mjs#L29

And If I'm reading it correctly it does:
1. add a task (100x)
2. check off a task (100x)
3. delete a task (100x)

File: resources/todomvc/readme.md (hunk 2 above)
Comment (flashdesignory) [reply resources/todomvc/readme.md:16]: yes, that's correct - I can add more clarity

File: resources/todomvc/readme.md (hunk 1 above)
Comment (flashdesignory) [reply resources/todomvc/readme.md]: @bgrins - your proposed change made it a lot clearer 😄

**Review by bgrins — APPROVED** [review]
